sea-orm-codegen/src/entity/writer.rs

```diff
@@ -818,16 +818,9 @@ impl EntityWriter {
     }
 
     pub fn gen_schema_name(schema_name: &Option<String>) -> Option<TokenStream> {
-        match schema_name {
-            Some(schema_name) => {
-                if schema_name != "public" {
-                    Some(quote! { #schema_name })
-                } else {
-                    None
-                }
-            }
-            None => None,
-        }
```

Comment:
Then we can remove this checking at codegen. This checking should be part of CLI anyway.